third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1083,11 +1083,62 @@ function isValidBaseURL(url) {
 
 )"""";
 
+// Script which sets a handler for collecting source maps from scripts in the
+// recording. Runs when recording/replaying if source map collection is enabled.
+const char* gReactDevtoolsScript = R""""(
+
+(() => {
+
+const hook = {
+  supportsFiber: true,
+  inject,
+  onCommitFiberUnmount,
+  onCommitFiberRoot,
+  onPostCommitFiberRoot,
+};
+
+Object.defineProperty(window, "__REACT_DEVTOOLS_GLOBAL_HOOK__", {
+  configurable: false,
+  enumerable: false,
```

Comment:
I think these are false by default, but if we're listing them we should include `writable: false` as well.

Reply:
Defining the property with just a getter should do this.  I cribbed this from the code which the react devtools backend uses to define the hook.